### ISSUE TITLE
Player selection menu doesn't open in Gnome 46.6 (Fedora 40)

### ISSUE THREAD

**Issue by mkalinski** [Moon-0xff/gnome-mpris-label#111]
Clicking on the label used to open a menu with player selection, but it no longer works.

Now clicking on the label produces the following error in logs:

```
gnome-shell[4113]: JS ERROR: Error: No field "red" on boxed type Color
                   _determineColors@file:///home/mk/.local/share/gnome-shell/extensions/mprisLabel@moon-0xff.github.com/extension.js:111:20
                   _buildMenu/<@file:///home/mk/.local/share/gnome-shell/extensions/mprisLabel@moon-0xff.github.com/extension.js:396:11
                   _buildMenu@file:///home/mk/.local/share/gnome-shell/extensions/mprisLabel@moon-0xff.github.com/extension.js:391:21
                   _activateAction@file:///home/mk/.local/share/gnome-shell/extensions/mprisLabel@moon-0xff.github.com/extension.js:275:10
                   _activateButtonAction@file:///home/mk/.local/share/gnome-shell/extensions/mprisLabel@moon-0xff.github.com/extension.js:253:8
                   _onClick/this._scheduledActionTimeout<@file:///home/mk/.local/share/gnome-shell/extensions/mprisLabel@moon-0xff.github.com/extension.js:172:10
                   @resource:///org/gnome/shell/ui/init.js:21:20
```

**Comment by Moon-0xff** [comment]
Hello @mkalinski,  
I was aware that the last changes to the code could be a problem for GNOME 45/46. I did amend the changes to assure the extension works in 45/46, but I didn't submit them to `extensions.gnome.org`

Can you install the development version of the extension following the [readme's instructions](https://github.com/Moon-0xff/gnome-mpris-label) and check if the menu works? If that's the case I just need to upload the amended version.

**Comment by mkalinski** [comment]
Yes, the menu works in the version from the `stable` branch.

**Comment by Moon-0xff** [comment]
~I also forgot to fast-forward the stable branch.~
I wanted the `main` branch tested, but I'll bet it's not necessary.  
I'll upload the amended version when I'm able. Please notify us if the extension stops working again.

**Comment by Moon-0xff** [comment]
Update is live! You probably received already. It reverts the problematic change that prevented the menu from appearing, and also adds a fix for plexamp(#110).

Thanks for raising this issue.
